src/main/java/com/minecolonies/coremod/network/messages/SchematicSaveMessage.java

```diff
@@ -0,0 +1,117 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.coremod.configuration.Configurations;
+import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.colony.Structures;
+import com.minecolonies.coremod.util.*;
+import com.minecolonies.structures.helpers.Structure;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessageHandler;
+import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Save Schematic Message.
+ */
+public class SchematicSaveMessage implements IMessage, IMessageHandler<SchematicSaveMessage, IMessage>
+{
+    private       byte[] data           = null;
+    final private int    MAX_TOTAL_SIZE = 32767;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add underscores to this numeric value for readability [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2148)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "MAX_TOTAL_SIZE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00116)